src/Microsoft.Dnx.Tooling/Building/BuildManager.cs

```diff
@@ -413,7 +413,7 @@ private static string Normalize(string projectDir)
 
         private static string GetPackagePath(Runtime.Project project, string outputPath, bool symbols = false)
         {
-            string fileName = project.Name + "." + project.Version + (symbols ? ".symbols" : "") + ".nupkg";
+            string fileName = project.Name + "." + project.Version.ToString() + (symbols ? ".symbols" : "") + ".nupkg";
```

Comment:
I guess it's Ok to make the code nicer here BTW in this PR:
1. Use string interpolation
2. Replace hard-coded `".nupkg"` with `NuGet.Constants.PackageExtension`